src/main/java/com/azure/spring/dev/tools/dependency/support/SpringProjectMetadataReader.java

```diff
@@ -43,9 +47,16 @@ public String getCurrentVersion() {
             .stream()
             .filter(p -> p.getReleaseStatus().equals(ReleaseStatus.GENERAL_AVAILABILITY))
             .filter(p -> p.getVersion().matches("2\\.\\d\\.\\d+"))
-            .filter(Objects::nonNull)
             .map(ProjectRelease::getVersion)
+            .map(this::parse)
+            .sorted(Comparator.reverseOrder())
```

Comment:
will this work?

Reply:
sure, `Version` implements `Comparable` interface and override `compareTo` method